src/modules/contract/contract.service.spec.ts

```
@@ -0,0 +1,108 @@
+  let humanReviewRepository: jest.Mocked<Repository<HumanReview>>;
+  let aiService: AiService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [
+        ContractService,
+        { provide: AiService, useValue: { analyzeContract: jest.fn(), answerQuestion: jest.fn() } },
+        { provide: getRepositoryToken(Contract), useValue: { create: jest.fn(), save: jest.fn(), findOne: jest.fn(), find: jest.fn(), remove: jest.fn() } },
+        { provide: getRepositoryToken(Clause), useValue: { save: jest.fn() } },
+        { provide: getRepositoryToken(RiskFlag), useValue: { findOne: jest.fn(), save: jest.fn() } },
+        { provide: getRepositoryToken(Summary), useValue: { } },
```

ESLint

[error] Delete `·`

(prettier/prettier)